301-multi-vmss-low-priority-autoscale/azuredeploy.json

```diff
@@ -0,0 +1,476 @@
+                "networkSecurityGroup": {
+                  "id": "[resourceId('Microsoft.Network/networkSecurityGroups', variables('networkSecurityGroupName'))]"
+              }
+            }
+          }
+          ]
+        }
+      },
+      {
+        "type": "Microsoft.Network/publicIPAddresses",
+        "name": "[variables('publicIPAddressName')]",
+        "location": "[variables('location')]",
+        "apiVersion": "2017-08-01",
+        "sku": {
+            "name": "Standard"
+        },
+        "properties": {
+            "publicIPAllocationMethod": "Static",
+            "dnsSettings": {
+                "domainNameLabel": "[variables('namingInfix')]"
+            }
+        }  
+      },
+      {
+        "type": "Microsoft.Network/loadBalancers",
+        "name": "[variables('loadBalancerName')]",
+        "location": "[variables('location')]",
+        "apiVersion": "2017-08-01",
+        "sku": {
+          "name": "Standard"
+        },
+        "dependsOn": [
+          "[concat('Microsoft.Network/publicIPAddresses/', variables('publicIPAddressName'))]"
+        ],
+        "properties": {
+          "frontendIPConfigurations": [
+            {
+              "name": "LoadBalancerFrontEnd",
+              "properties": {
+                "publicIPAddress": {
+                  "id": "[variables('publicIPAddressID')]"
+                }
+              }
+            }
+          ],
+          "backendAddressPools": [
+            {
+              "name": "[variables('bePoolName')]"
+            }
+          ],
+          "inboundNatPools": [
+            {
+              "name": "[variables('lpNatPoolName')]",
+              "properties": {
+                "frontendIPConfiguration": {
+                  "id": "[variables('frontEndIPConfigID')]"
+                },
+                "protocol": "tcp",
+                "frontendPortRangeStart": "[variables('lpNatStartPort')]",
+                "frontendPortRangeEnd": "[variables('lpNatEndPort')]",
+                "backendPort": "[variables('natBackendPort')]"
+              }
+            },
+            {
+              "name": "[variables('regNatPoolName')]",
+              "properties": {
+                "frontendIPConfiguration": {
+                  "id": "[variables('frontEndIPConfigID')]"
+                },
+                "protocol": "tcp",
+                "frontendPortRangeStart": "[variables('regNatStartPort')]",
+                "frontendPortRangeEnd": "[variables('regNatEndPort')]",
+                "backendPort": "[variables('natBackendPort')]"
+              }
+            }                  
+          ],
+          "loadBalancingRules": [
+            {
+              "name": "httpRule",
+              "properties": {
+                "frontendIPConfiguration": {
+                  "id": "[concat(resourceId('Microsoft.Network/loadBalancers', variables('loadBalancerName')), '/frontendIPConfigurations/LoadBalancerFrontEnd')]"
```

Comment:
remove concat and just use resourceId - same as in the other PR (won't comment on each instance)